lessons/Early-Math/distributive-property/langs/en-us/index.adoc

```diff
@@ -40,69 +40,103 @@ Students develop an understanding of the @vocab{Distributive Property} by examin
+@teacher{Facilitate a discussion where students think deeply about what is happening in the scenarios above, namely, that distribution allows us to rewrite a product as a sum or a sum as a product.}
+
+@slidebreak
 
 @lesson-point{
 Multiplying the sum of two addends by a number produces the same result as multiplying _each_ addend by that number before finding the sum.
 }
 
+@slidebreak
 
+@teacher{
 In the next activity, students will use Circles of Evaluation to explore how the mathematical structure of an expression shifts when we apply Distributive Property.
+}
 
 @lesson-instruction{
 - On @printable-exercise{pages/sum-to-product.adoc}, fill in the blanks so that the Circle of Evaluation on the right is equivalent to the Circle of Evaluation on the left.
 - Use computation to confirm that the Circles of Evaluation in each pair are equivalent.
-- After you finish evaluating each Circle of Evaluation, place a checkmark next to the Circle of Evaluation that you think results in a simpler comuptation.
-- When was it easier to compute the _sum_? When was it easier to compute the _product_?
-  * _There are no right or wrong answers, here. We want students to begin to consider how a nuanced understanding of the Distributive Property can result in more flexible and efficient mental math._
+- After you finish evaluating each Circle of Evaluation, place a checkmark next to the Circle of Evaluation that you think results in a simpler computation.
+}
+
+@QandA{
+@Q{When was it easier to compute the _sum_? When was it easier to compute the _product_?}
+@A{There are no right or wrong answers, here. We want students to begin to consider how a nuanced understanding of the Distributive Property can result in more flexible and efficient mental math.}
+}
+
+
+@slidebreak
+
+@lesson-instruction{
 - On @printable-exercise{pages/product-to-sum.adoc}, the Circle of Evaluation representing a product is provided, and you will complete the Circle of Evaluation that represents a sum.
-- Try @printable-exercise{pages/distribution-challenge.adoc}, where you will practice applying the Distributive Property in both directions - and with expressions that include fractions and decimals!
+- Complete @printable-exercise{pages/distribution-challenge.adoc}, where you will practice applying the Distributive Property in both directions - and with expressions that include fractions and decimals!
 }
 
+@teacher{The last two questions on each of the above pages ask students to use a made-up function at the top of the Circle of Evaluation, rather than an operator. These functions - which are descriptively named `triple`, `quadruple`, `times9`, and `times7` - take in just _one_ input. Students should observe that these functions distribute over addition and subtraction in the same manner as multiplication does.}
```

Comment:
Small nitpick, but I'm guessing `quadruple` is going to give some kids a hard time. Maybe replace it would `double`?